src/main/java/games/strategy/net/IServerMessenger.java

```diff
@@ -35,23 +35,23 @@
    */
   Set<INode> getNodes();
 
-  void NotifyIPMiniBanningOfPlayer(String ip, Date expires);
+  void notifyIPMiniBanningOfPlayer(String ip, Instant expires);
 
-  void NotifyMacMiniBanningOfPlayer(String mac, Date expires);
+  void notifyMacMiniBanningOfPlayer(String mac, Instant expires);
 
-  void NotifyUsernameMiniBanningOfPlayer(String username, Date expires);
+  void notifyUsernameMiniBanningOfPlayer(String username, Instant expires);
 
   String getPlayerMac(String name);
 
-  void NotifyUsernameMutingOfPlayer(String username, Date muteExpires);
+  void notifyUsernameMutingOfPlayer(String username, Instant muteExpires);
 
-  void NotifyIPMutingOfPlayer(String ip, Date muteExpires);
+  void notifyIPMutingOfPlayer(String ip, Instant muteExpires);
```

Comment:
Since you already renamed this method, can you please change `IP` to `Ip` to remove the remaining Checkstyle warning on this line?